script/import_scripts/base.rb

```diff
@@ -444,7 +444,7 @@ def create_categories(results)
       else
         # Basic massaging on the category name
         params[:name] = "Blank" if params[:name].blank?
-        params[:name].strip!
+        params[:name].dup.strip!
```

Comment:
Since we’re duplicating here, the original `params[:name]` (which gets used below) is never actually being stripped.

Perhaps something like this might work?

```suggestion
        params[:name] = params[:name].strip
```